gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Wed May 29 09:07:44 CDT 2019
+#Sun Mar 14 14:43:20 CDT 2021
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.1.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-bin.zip
```

Comment:
Good questions. So I just updated to whatever Android Studio wanted to in order to update to the latest AGP for my Android Studio version at the time. I have no idea how it chooses this.

Reply:
Discussed offline, most likely it chose the minimum required version for AGP `4.1.2`, and the `bin` thing is a strange inconsistency since it also gives you the hint to change it to `all`.